### DISCUSSION TITLE
Condense melon slices to melons

### DISCUSSION THREAD

**Discussion by untuned** [EssentialsX/Essentials#4135] [Ideas and Feature Suggestions] [answered]
I would like to be able to use `/condense` to turn melon slices into melons. It is around the same process as turning redstone into a block of redstone, so I don't imagine it would be too difficult to implement.

**Comment by pop4959** [comment, marked as answer]
Condense only works with reversible recipes. Since you can't turn a melon back into the same number of melon slices, it is not possible to condense them.
